source/git-parser.js

```diff
@@ -291,7 +291,7 @@ const updatePatchHeader = (result, lastHeaderIndex, ignoredDiffCountTotal, ignor
 }
 
 exports.parsePatchDiffResult = (patchLineList, text) => {
-  if (!text) return {};
+  if (!text) return '';
```

Comment:
`parsePatchDiffResult` returns a string so if this error case, an empty string should be returned instead of an empty object for consistency

Reply:
Nice, I think we should either line 352 to `''` or this to `null`.

Reply:
Good catch :+1: I'll go for `null` here too then